tests/spec/SpecHelper.js

```diff
@@ -171,9 +172,16 @@ function makeDefaultBody() {
   return "<section id='sotd'><p>foo</p></section><section id='toc'></section>";
 }
 
-function makeStandardOps() {
+/**
+ *
+ * @param configParams
+ * @param bodyParams
+ * @returns {{config: {editors, specStatus, edDraftURI, shortName, previousMaturity, previousPublishDate, errata, implementationReportURI, perEnd, lint} & any, body: string}}
+ */
+
+function makeStandardOps(configParams, bodyParams = makeDefaultBody()) {
   return {
-    config: makeBasicConfig(),
-    body: makeDefaultBody(),
+    config: Object.assign(makeBasicConfig(),configParams),
```

Comment:
Please use the newer syntax I provided tho :) No need to use object.assign.

Reply:
Oops Ok :sweat_smile: